cassandane/Cassandane/Cyrus/Master.pm

```diff
@@ -1399,4 +1406,54 @@ sub test_daemon_exits
                                  qr{too many failures for});
 }
 
+sub test_babysit
+{
+    my ($self) = @_;
+
+    $self->{instance}->{no_check_syslog} = 1;
+
```

Comment:
There's a magic attribute for this already, something like this should do:

```suggestion
sub test_babysit
    :NoCheckSyslog
{
    my ($self) = @_;

```

Though maybe you tried that already and it _didn't_ work, which would be very interesting to hear about

Reply:
Ah! I had found another test use no_check_syslog => 1 in stop() so figured that was the state of the art. Thank you, fixed

Reply:
Ahh yep.  Useful rule of thumb: any time you need to do something unusual at the start of a test, have a look at Cassandane::Cyrus::TestCase and see if there's already a magic for it :)